py/desimeter/posparams/fitter.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+"""
+Best-fit calculation module for positioner calibration parameters.
+"""
+
+import sys
+import math
+import scipy.optimize
+
+# DESI-specific imports
+# path handling is to be improved as I migrate better into github/desimeter
+petal_path = 'C:/Users/joe/Desktop/desi_svn/focalplane/plate_control/trunk/petal/'
+sys.path.append(petal_path)
```

Comment:
See earlier comments about dependencies and avoiding hardcoded paths.

Reply:
We need to resolve whether I will write a desimeter-specific verison of postransforms. If not, then I need some access to code on the SVN. Is there any standard method in desihub to import SVN code?

Reply:
To close this out --- after discussion we did decide to fork. See PR #56